library/kotlin/src/io/envoyproxy/envoymobile/EnvoyClient.kt

```diff
@@ -0,0 +1,37 @@
+package io.envoyproxy.envoymobile
```

Comment:
File was renamed from `Envoy`, but git didn't recognize this as a rename because there were enough changes in the file